some italian and portugese test are failing. I dont understand what this error is for ?
 1 failing

  1) Check challenges
       Language: italian
         SuperBlock: 2022/responsive-web-design
           Block: learn-accessibility-by-building-a-quiz
             Title: Step 67
               ID: 6148e41c728f65138addf9cc
                 Check tests against solutions
                   Solution 1 must pass the tests:

      Check next step for solution!
Test text: <p>Dovresti avere una regola <code>@media (prefers-reduced-motion: no-preference)</code>.</p>
      + expected - actual

      -0
      +1
      
  AssertionError: expected +0 to equal 1
      at eval (eval at <anonymous> (http://127.0.0.1:8080/js/frame-runner-543042f22803f54627ea.js:2:123872), <anonymous>:1:8)
      at HTMLDocument.<anonymous> (http://127.0.0.1:8080/js/frame-runner-543042f22803f54627ea.js:2:123872)
      at u (http://127.0.0.1:8080/js/frame-runner-543042f22803f54627ea.js:2:26888)
      at c (http://127.0.0.1:8080/js/frame-runner-543042f22803f54627ea.js:2:27190)

--- a/curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md
+++ b/curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md
@@ -39,7 +39,7 @@ Work with both dates and time.
 
 #### --text--
 
-What is the easiest way to create a new `Date` object instance?
+Which code below creates a new `Date` object instance?
 
 #### --distractors--
 

--- a/curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md
+++ b/curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md
@@ -105,7 +105,7 @@ To get the current year we use which instance method of the `Date` object?
 
 #### --text--
 
-To get the current month(0-11) we use which instance method of the `Date` object?
+To get the current month (0 - 11) we use which instance method of the `Date` object?
 
 #### --distractors--
 
@@ -127,7 +127,7 @@ To get the current month(0-11) we use which instance method of the `Date` object
 
 #### --text--
 
-The built in javascript method that allows us to format a date as a string is?
+The built-in JavaScript method that allows us to format a date as a string is?
 
 #### --distractors--
 
@@ -149,51 +149,55 @@ The built in javascript method that allows us to format a date as a string is?
 
 #### --text--
 
-It is the year 2022 this code `console.log(new Date().getFullYear());` outputs what ?
+What would output `console.log(new Date().getFullYear());`, if it would be the year `2022`?
 
 #### --distractors--
 
-June
+`"June"`
 
 ---
 
-1960
+`1960`
 
 ---
 
-2008
+`2008`
 
 #### --answer--
 
-2022
+`2022`
 
 ### --question--
 
 #### --text--
 
-If today's date was 18th of June, 2023. Which line of code below would give us the output `6/18/2023`
+What will be the output of the following code?
+
+```js
+console.log(new Date("2023-06-18T12:00:00").toUTCString());
+```
 
 #### --distractors--
 
-`console.log('new Date().toLocaleDateString()');`
+`Mon, 19 Jun 2023 12:00:00 GMT`
 
 ---
 
-`console.log(new Date().toCityDateString());`
+`Sun, 18 Jun 2023 12:00:00 GMT+1`
 
 ---
 
-`console.log(new Date().toLocaleTimeString());`
+`Sun, 18 Jun 2023 13:00:00 GMT`
 
 #### --answer--
 
-`console.log(new Date().toLocaleDateString());`
+`Sun, 18 Jun 2023 12:00:00 GMT`
 
 ### --question--
 
 #### --text--
 
-If the current time was 1:00pm. Which line of code below would give us the output `1:00:00 PM`
+If the time in your locale is formatted as `HH:MM:SS AM/PM`, which line of code would correctly display the current time?
 
 #### --distractors--
 
@@ -223,11 +227,11 @@ A USA-863 string format.
 
 ---
 
-An MP3 string format
+An MP3 string format.
 
 ---
 
-A UTC string format
+A UTC string format.
 
 #### --answer--
 
@@ -241,19 +245,19 @@ What should an ISO-8601 date format look like?
 
 #### --distractors--
 
-DAY-MONTH-YEAR
+DD-MM-YYYY
 
 ---
 
-DAY-MONTH-YEAR:HOUR
+DD-MM-YYTHH
 
 ---
 
-DAY-MONTH-YEAR HOUR:MINUTES:SECONDS
+YYYY-MM-DDTHH:mm:ssZ
 
 #### --answer--
 
-YEAR-MONTH-DAY HOUR:MINUTES:SECONDS:MILLISECONDS
+YYYY-MM-DDTHH:mm:ss.sssZ
 
 ### --question--
 
@@ -263,19 +267,19 @@ What is the output of `console.log(new Date(2003, 6, 27).getMonth());`?
 
 #### --distractors--
 
-27/6/2003
+`"27/6/2003"`
 
 ---
 
-27
+`27`
 
 ---
 
-2003
+`2003`
 
 #### --answer--
 
-6
+`6`
 
 ### --question--
 
@@ -303,23 +307,27 @@ How would you format a date to a locale-specific string or a more readable forma
 
 #### --text--
 
-The `.toLocaleDateString()` date format looks like?
+Which option will correctly create a `Date` object representing January 1, 2025?
+
+```js
+const d = ?;
+```
 
 #### --distractors--
 
-MONTH / DAY / SECONDS
+`const d = new Date("1-1-2025");`
 
 ---
 
-MONTH / DAY
+`const d = new Date(2025, 0, 0);`
 
 ---
 
-YEAR / MONTH / DAY
+`const d = new Date("2025-01-01T00:00:00");`
 
 #### --answer--
 
-MONTH / DAY / YEAR
+`const d = new Date("2025-01-01");`
 
 ### --question--
 
@@ -329,19 +337,19 @@ What does `new Date().getDay()` return?
 
 #### --distractors--
 
-It returns the current day of the week as a string
+It returns the current day of the week as a string. Starting with Friday as `0`.
 
 ---
 
-It returns the current day of the year as a boolean.
+It returns the current day of the year as a number. Starting with Monday as `0`.
 
 ---
 
-It returns the current day of the month as a decimal.
+It returns the current day of the month as a decimal. Starting with Sunday as `1`.
 
 #### --answer--
 
-It returns the current day of the week as a number.
+It returns the current day of the week as a number. Starting with Sunday as `0`.
 
 ### --question--
 
@@ -351,19 +359,19 @@ What is the output of `console.log(new Date(2003, 6, 27).getFullYear());`?
 
 #### --distractors--
 
-27/6/2003
+`"27/6/2003"`
 
 ---
 
-27
+`27`
 
 ---
 
-6
+`6`
 
 #### --answer--
 
-2003
+`2003`
 
 ### --question--
 
@@ -391,65 +399,73 @@ A UTC string format
 
 #### --text--
 
-If we are in the month of October this code `console.log(new Date().getMonth());` outputs what ?
+If we are in the month of October, what will `console.log(new Date().getMonth());` output?
 
 #### --distractors--
 
-1
+`1`
 
 ---
 
-3
+`3`
 
 ---
 
-10
+`10`
 
 #### --answer--
 
-9
+`9`
 
 ### --question--
 
 #### --text--
 
-If the ouput of this code is "2021", what date method is missing from `console.log(new Date("2021-12-25")(MISSING METHOD))`;
+Which option will output `"2021"` for the following object?
+
+```js
+const d = new Date("2021-12-25");
+```
 
 #### --distractors--
 
-`.getTime()`
+`console.log(d.getTime())`
 
 ---
 
-`.toUTCString()`
+`console.log(d.toUTCString())`
 
 ---
 
-`.getDate()`
+`console.log(d.getDate())`
 
 #### --answer--
 
-`.getFullYear()`
+`console.log(d.getFullYear())`
 
 ### --question--
 
 #### --text--
 
-If the ouput of this code is "2021-12-25T00:00:00.000Z", what date method is missing from `console.log(new Date("2021-12-25")(MISSING METHOD))`;
+Which option will output `"2021-12-25T00:00:00.000Z"` for the following object?
+
+```js
+const d = new Date("2021-12-25");
+```
 
 #### --distractors--
 
-`.getFullYear()`
+`console.log(d.getFullYear())`
 
 ---
 
-`.toUTCString()`
+`console.log(d.toUTCString())`
 
 ---
 
-`.getTime()`
+`console.log(d.getTime())`
 
 #### --answer--
 
-`.toISOString()`
+`console.log(d.toISOString())`
 

--- a/curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md
+++ b/curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md
@@ -47,11 +47,11 @@ What is the easiest way to create a new `Date` object instance?
 
 ---
 
-`Date.appear()`
+`Date.fetch()`
 
 ---
 
-`Date new()`
+`new.Date()`
 
 #### --answer--
 
@@ -223,7 +223,7 @@ A USA-863 string format.
 
 ---
 
-An mp3 string format
+An MP3 string format
 
 ---
 
@@ -241,19 +241,19 @@ What should an ISO-8601 date format look like?
 
 #### --distractors--
 
-`DAY`-`MONTH`-`YEAR`
+DAY-MONTH-YEAR
 
 ---
 
-`DAY`-`MONTH`-`YEAR`:`HOUR`
+DAY-MONTH-YEAR:HOUR
 
 ---
 
-`DAY`-`MONTH`-`YEAR` `HOUR`:`MINUTES`:`SECONDS`
+DAY-MONTH-YEAR HOUR:MINUTES:SECONDS
 
 #### --answer--
 
-`YEAR`-`MONTH`-`DAY` `HOUR`:`MINUTES`:`SECONDS`:`MILLISECONDS`
+YEAR-MONTH-DAY HOUR:MINUTES:SECONDS:MILLISECONDS
 
 ### --question--
 
@@ -307,19 +307,19 @@ The `.toLocaleDateString()` date format looks like?
 
 #### --distractors--
 
-`MONTH`/`DAY`/`SECONDS`
+MONTH / DAY / SECONDS
 
 ---
 
-`MONTH`/`DAY`
+MONTH / DAY
 
 ---
 
-`YEAR`/`MONTH`/`DAY`
+YEAR / MONTH / DAY
 
 #### --answer--
 
-`MONTH`/`DAY`/`YEAR`
+MONTH / DAY / YEAR
 
 ### --question--
 
@@ -377,7 +377,7 @@ A door dash string format
 
 ---
 
-An mp3 string format
+An MP3 string format
 
 ---
 

